cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackImageVerifier.java

```diff
@@ -17,18 +18,27 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(OpenStackImageVerifier.class);
 
-    public void exist(OSClient<?> osClient, String name) {
+    public boolean exist(OSClient<?> osClient, String name) {
+        boolean exist = false;
+        if (getStatus(osClient, name) != null) {
+            exist = true;
+        }
+        return exist;
+    }
+
+    public ImageStatus getStatus(OSClient<?> osClient, String name) {
+        ImageStatus imageStatus;
         List<? extends Image> images = osClient.imagesV2().list(Collections.singletonMap("name", name));
         if (images == null || images.isEmpty()) {
+            imageStatus = null;
```

Comment:
the ImageStatus is null since the first declaration two lines above

Reply:
that is just declaration, method local vars are not initialized to null